libfreerdp/core/metrics.c

```diff
@@ -30,8 +30,10 @@ double metrics_write_bytes(rdpMetrics* metrics, UINT32 UncompressedBytes, UINT32
 	metrics->TotalUncompressedBytes += UncompressedBytes;
 	metrics->TotalCompressedBytes += CompressedBytes;
 
-	CompressionRatio = ((double) CompressedBytes) / ((double) UncompressedBytes);
-	metrics->TotalCompressionRatio = ((double) metrics->TotalCompressedBytes) / ((double) metrics->TotalUncompressedBytes);
+	if (UncompressedBytes != 0)
+		CompressionRatio = ((double) CompressedBytes) / ((double) UncompressedBytes);
```

Comment:
Maybe it's redundancy comment: Shouldn't we initialize 'CompressionRatio' since it's possible that we never set it?